tests/StorageApiTestCase.php

```diff
@@ -349,15 +351,17 @@ protected function getTestBucketId($stage = self::STAGE_IN)
         return $this->_bucketIds[$stage];
     }
 
-    protected function createAndWaitForEvent(\Keboola\StorageApi\Event $event)
+    protected function createAndWaitForEvent(Event $event, Client $sapiClient = null)
```

Comment:
Není to tak dávno, co jsem odstraňoval typehinty, aby to fungovalo na php 5.6. Tohle issue tady nenastane ?

Reply:
Ja myslim ze ne. Nemam tam`?Client $sapiClient = null`

Reply:
ok, zkusil jsem to http://sandbox.onlinephpfunctions.com/code/fadbdafb644c16ff55c171776fa2d54ce0a59054 a funguje to. Myslel jsem, že datový typy jsou až od php7, ale tyká se to jenom return type